### PR TITLE
Misc Fixes (1.8.2)

### PR DESCRIPTION
**Pull request by mlzummo**
Fixes GH-1232
Fixes GH-1233

Signed-off-by: Mathew Zummo <mzummo@gmail.com>

### CODE REVIEW

**Comment by mlzummo** [comment]
@GordonSmith the only thing i dont like about these fixes so far and my update to the color stuff is the hint of a vertical line at the data points now sure what is up with that....

http://rawgit.com/mlzummo/Visualization/AMCHARTS_COLOR_FIX_3_1.8/demos/dermatology.html?src/amchart/Area

**Comment by mlzummo** [comment]
might not be working 100% yet

File: src/amchart/Column.js [hunk 1]
```diff
@@ -23,7 +23,7 @@
     Column.prototype.publish("stacked", false, "boolean", "Stack Chart",null,{tags:["Basic","Shared"]});
     Column.prototype.publish("fillOpacity", 0.7, "number", "Opacity of The Fill Color", null, {min:0,max:1,step:0.001,inputType:"range",tags:["Intermediate","Shared"]});
 
-    Column.prototype.publish("paletteGrouping", "By Column", "set", "Palette Grouping",["By Category","By Column"],{tags:["Basic"]});
+    //Column.prototype.publish("paletteGrouping", "By Column", "set", "Palette Grouping",["By Category","By Column"],{tags:["Basic"]});
```
Comment (GordonSmith) [inline src/amchart/Column.js]: Remove commented code

File: src/amchart/Column.js [hunk 2]
```diff
@@ -51,35 +58,9 @@
...
-        this._chart.categoryAxis.startOnAxis = false; //override due to render issue
-
-        if (this._rangeType === "candle-ohlc") {
-            this._gType = this.useOhlcLines() ? "ohlc" : "candlestick";
-        } else {
-            this._gType = "column";
-        }
-
-        this.buildGraphs(this._gType);
+        this._chart.categoryAxis.startOnAxis = false; //override due to render issue ?? does this stole hold true?
```
Comment (GordonSmith) [inline src/amchart/Column.js]: Does this need a re-test to see if its needed?  If so comment should be updated.

File: src/amchart/CommonSerial.js [hunk 3]
```diff
@@ -125,6 +129,11 @@
 
     CommonSerial.prototype.publish("equalSpacing", false, "boolean", "Show Chart Scrollbar",null,{tags:["Intermediate","Shared"]});
 
+
+    CommonSerial.prototype.publish("paletteGrouping", "By Column", "set", "Palette Grouping",["By Category","By Column"],{tags:["Basic"]});
+
+
+
```
Comment (GordonSmith) [inline src/amchart/CommonSerial.js]: blank lines.

**Comment by GordonSmith** [comment]
Minor comments let me know when you want me to pull...

**Comment by mlzummo** [comment]
@GordonSmith ready to pull

**Pull request merged**